src/Raven.Server/Rachis/Follower.cs

```diff
@@ -65,7 +67,7 @@ private void FollowerSteadyState()
 
                         return;
                     }
-
+                    _debugRecorder.Record($"Got {appendEntries.EntriesCount} entries");
```

Comment:
This is going to be running all the time, and it is going to always be allocating strings that we'll most of the time never look at.